crates/sui-faucet/src/main.rs

```diff
@@ -30,10 +31,9 @@ use uuid::Uuid;
 
 const CONCURRENCY_LIMIT: usize = 30;
 
-struct AppState<F = SimpleFaucet> {
+struct AppState<F = Arc<SimpleFaucet>> {
     faucet: F,
     config: FaucetConfig,
-    // TODO: add counter
```

Comment:
What was this in relation to, and why is it gone now?

Reply:
It was originally an implementation on top of the WAL where if we retried too many times but still failed I would just drop the item from the WAL. This was designed for slightly better recovery

Reply:
Got it, thanks!